code-challenges.js

```diff
@@ -32,6 +68,83 @@ var hand4 = [3, 5, 3, 5, 3]
 // Expected output: true
 
 
+//the fucnction is going to take in an array of 5 numbers equal to a full full house which is a pair and three of a kind
+//Im thinking that there will be some type of conditonal statment saying that if there are so many instances of a value then perform this action
+//most likely going to be a map
+//if 3 values are the same in different indexes and 2 other values match the other indexs
+
+
+//an array counter method can work to check how many times an instance occurs!!!
+//
+//
+// const fullHouse = (array) =>{
+//   for(let i = 0; i < array.length; i++) {
+//     if(array[i] === array[i] && array[i] === array[i] === array[i]){
+//       return true
+//   }else{
+//     return false
+// }
+// }
+// }
+
+
+//other idea would be to sort them by order and compare first three with last three since it would only work if they were all the same
+
+const fullHouse = (array) =>{
+  //sorting the values willmake sure that like values stay next to each other.
+let sortedArray = array.sort((a, b) => a - b)
+//compare three equal values at the start of array and two equal values in the back
```

Comment:
This is an example of a comment getting really close to describing what it's trying to do. To improve this comment, relate the need for the kind of check to what a full house is.